### PR TITLE
Cli: Constants

### PR DESCRIPTION
**Pull request by estebanprimost**
Move constants to `constants.js`

### CODE REVIEW

File: packages/cli/src/commands/status.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+const { cli } = require('cli-ux')
+const humanizeDuration = require('humanize-duration')
+const prettyBytes = require('pretty-bytes')
+const { flags } = require('@oclif/command')
+
+const BaseCommand = require('../base-command')
+const { pm2Connect, pm2List, pm2Disconnect, pm2Describe } = require('../pm2-async')
+const { SEEDER_DAEMON } = require('../constants')
+
+class StatusCommand extends BaseCommand {
```
Comment (dpaez) [inline packages/cli/src/commands/status.js]: I think this belongs to a different PR.

Comment (estebanprimost) [reply packages/cli/src/commands/status.js]: Yes, this is form #72
